pkg/controllers/dynakube/extension/consts/consts.go

```diff
@@ -20,11 +20,23 @@ const (
 	ExtensionsCollectorComPort        = 14599
 	ExtensionsCollectorTargetPortName = "collector-com"
 
-	ExtensionsCustomTlsCertificate = "custom-tls-certificates"
+	ExtensionsTlsSecretSuffix = "-extensions-controller-tls"
 
 	// TLSKeyDataName is the key used to store a TLS private key in the secret's data field.
 	TLSKeyDataName = "tls.key"
 
 	// TLSCrtDataName is the key used to store a TLS certificate in the secret's data field.
 	TLSCrtDataName = "tls.crt"
 )
+
+func GetCertificateAltNames(dkName string) []string {
```

Comment:
please don't put functions in constants files. create new file or package, similar I force to do in @wepudt  PR.

Reply:
Fixed